docs/api/data.rst

```diff
@@ -48,6 +48,42 @@ It provides a set of 25,000 highly polar movie reviews for training, 25,000 for
 
     IMDB
 
+Sentiment Analysis: MR
+~~~~~~~~~~~~~~~~~~~~~~~~
+`MR <https://www.cs.cornell.edu/people/pabo/movie-review-data/>`_ is a data set of movie reviews for sentiment analysis. Available are collections of movie-review documents labeled with respect to their overall sentiment polarity (positive or negative).
+
+.. autosummary::
+    :nosignatures:
+
+    MR
+    
+Sentiment Analysis: SST
+~~~~~~~~~~~~~~~~~~~~~~~~
+`SST <http://nlp.stanford.edu/sentiment/>`_ is an extension of the MR data set. However, train/dev/test splits are provided and labels are fine-grained (very positive, positive, neutral, negative, very negative).
```

Comment:
Append the following: 
The training/development/test data sets have XXX, YYY, ZZZ sentences respectively. Note that some papers remove neutral sentences and consider binary sentiment polarity only.